src/app/plugins/bubblewrap.plugin.ts

```diff
@@ -13,7 +13,7 @@ export class BubbleWrapPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args: string[]) {
-    const wrap: string = "||pop||||pop||||pop||||pop||||pop||\n".repeat(5);
+    const wrap: string = "_\n_" + "||pop||||pop||||pop||||pop||||pop||\n".repeat(5);
```

Comment:
Maybe `"Here is some bubblewrap\n" + "...` so that there isnt weird blank space when not in compact mode